src/main/java/bowling_step3/controller/BowlingApp.java

```diff
@@ -0,0 +1,24 @@
+package bowling_step3.controller;
+
+import bowling_step3.domain.*;
+import bowling_step3.view.Input;
+import bowling_step3.view.Output;
+
+import java.util.stream.Stream;
+
+public class BowlingApp {
+    public static void main(String[] args) {
+        String name = Input.scanPlayer();
+        Player player = new Player(name);
+
+        Frames frames = Frames.create();
+        Frame frame =frames.first();
+        while (!(frame.next() == null && frame.done())) {
```

Comment:
```suggestion
        while (!frame.finished()) {
```

위와 같이 frame이 게임 종료 여부를 판단하도록 하는 것은 어떨까?